source/SIL.AppBuilder.Portal.Frontend/src/ui/components/header/notifications/data.tsx

```diff
@@ -58,7 +51,7 @@ export function withData(WrappedComponent) {
     render() {
       const dataProps = {
         haveAllNotificationsBeenSeen: this.haveAllNotificationsBeenSeen(),
-        isThereAtLeastOneNotificationToShow: this.isThereAtLeastOneNotificationToShow(),
+        isThereAtLeastOneNotificationToShow: true,
```

Comment:
was this just for testing purposes?

Reply:
need to fix up all of those... methods... good catch.